src/Effectful/Log.hs

```diff
@@ -46,6 +47,15 @@ runLog component logger maxLogLevel = evalStaticRep $ Log LoggerEnv
   , leMaxLogLevel = maxLogLevel
   }
 
+-- | Run the 'Log' effect with full logging options
+runLogWithEnv
```

Comment:
Maybe define `runLog` in terms of `runLogWithEnv`? I.e. `runLog component logger maxLogLevel = runLogWithEnv LoggerEnv ...`.

Reply:
:+1: